src/Message/IPNCallback.php

```diff
@@ -107,7 +107,7 @@ public function isCharge()
 
     /**
      * Is this a AUTH_ONLY_SINGLE_CHARGE IPN? This IPN is fired when
-     * a zero dollar transaction is made.
+     * a zero dollar transaction is made for standard subscription.
```

Comment:
What's a standard subscription? I assumed it meant non-trial but then below you say "standard subscription with trial" - do we support any subscriptions that are not standard?

Reply:
@kaylafuchs @jcmanzo I will include a link but basically they have multiple plan types.  The one we are using is standard subscription.  They are recommending standard subscription with trial.